api/server_test.go

```diff
@@ -71,7 +70,7 @@ func TestSocket(t *testing.T) {
 	}
 
 	t.Run("socket doesn't exist before", func(t *testing.T) {
-		tmpDir, err := ioutil.TempDir("", "testsocket")
+		tmpDir, err := os.MkdirTemp("", "testsocket")
```

Comment:
It's better to use `t.TempDir` so it's automatically removed at the end of the current test https://pkg.go.dev/testing@go1.23.1#T.TempDir

Reply:
Thanks, I completely forgot about this function, makes total sense, will update the code.